internal/network/command_transit_gateway_attachment.go

```diff
@@ -0,0 +1,130 @@
+package network
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	networkingv1 "github.com/confluentinc/ccloud-sdk-go-v2/networking/v1"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/errors"
+	"github.com/confluentinc/cli/v3/pkg/output"
+)
+
+type transitGatewayAttachmentCommand struct {
+	*pcmd.AuthenticatedCLICommand
+}
+
+type transitGatewayAttachmentHumanOut struct {
+	Id                         string `human:"ID"`
+	Name                       string `human:"Name"`
+	NetworkId                  string `human:"Network ID"`
+	RamShareArn                string `human:"RAM Share ARN"`
```

Comment:
this field is actually aws specific. let's rename it to `AwsRamShareArn`

Reply:
I left out the cloud provider prefix for all `transit_gateway_attachment` commands because I had the impression that we only support AWS Transit Gateway at the moment.

Reply:
Transit gateway is only an AWS construct so I think cloud is implied

Reply:
I can add the `AWS` prefix to these AWS specific resources to keep things consistent with what we have for `peering describe`. @ejagade @kabraham13 Is that okay?

I also notice that the UI specifies "AWS" as well, except for `RAM Share ID`. Just another reason to add the prefix there.
![image](https://github.com/confluentinc/cli/assets/135164550/3f1884be-7b04-479f-81c1-d13312069025)

If we want to proceed with adding the prefix for aws specific resources in `describe` and `list`, do we need it for the flags of `create`? (i.e. use `aws-ram-share-arn` instead of `--ram-share-arn`)

Reply:
the term "ram-share-arn" is very AWS specific. It won't cause confusion if the `aws` prefix is not there. But adding `aws` prefix shows the intent that it is AWS generated ID (similar to `vpc` or `VNet`)

Reply:
my vote is for `--aws-ram-share-arn`. let' confirm with @kabraham13 too

Reply:
I am okay with it

Reply:
Thanks for the confirmation. Updating the describe fields in https://github.com/confluentinc/cli/pull/2244, and will include the `aws` prefix for the `create` flags`.